frontend/src/screens/MainScreenWrapper.css

```diff
@@ -0,0 +1,52 @@
+html, body {
+    height: 100%;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #0C2D48;
+    font-family: 'Hiragino Sans', 'Hiragino Kaku Gothic ProN', Meiryo, sans-serif;
+  }
+  
+  .filter-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 20px;
+    background: #B1D4E0;
+    border-radius: 8px;
+    box-shadow: 0 2px 4px rgba(0,0,0,0.1); /* shadow */
+  }
+  
+  .filter-item + .filter-item {
+    margin-top: 20px; /* space between filter items */
+  }
+  
+  label {
+    margin-bottom: 5px;
+    font-weight: bold;
+  }
+  
+  select {
+    padding: 8px;
+    border: 1px solid #2E8BC0;
+    border-radius: 4px;
+    width: 100%; /* fill the container width */
```

Comment:
We could make all selects have a full width if we add `.filter-item { width: 100%; }`